batch/batch/front_end/front_end.py

```diff
@@ -1889,6 +1894,59 @@ async def get_job(request: web.Request, _, batch_id: int) -> web.Response:
     return json_response(status)
 
 
+@routes.get('/api/v1alpha/batches/{batch_id}/jobs/{job_id}/resource_usage')
+@billing_project_users_only()
+async def get_job_resource_usage(request: web.Request, _, batch_id: int) -> web.Response:
+    """
+    Get the resource_usage data for a job. The data is returned as a JSON object
+    transformed from a pandas DataFrame using the 'split' orientation.
+
+    Returns
+    -------
+    Example response:
+    {
+        // eg: input, main, output
+        "[job_stage]": {
+            "columns":[
+                "time_msecs",
+                "memory_in_bytes",
+                "cpu_usage",
+                "non_io_storage_in_bytes",
+                "io_storage_in_bytes",
+                "network_bandwidth_upload_in_bytes_per_second",
+                "network_bandwidth_download_in_bytes_per_second"
+            ],
+            "index":[0, 1, ...],
+            "data": [[<records>]],
+        }, ...
+    }
+    """
+
+    # pull this out separately as billing_project_users_only() does a permission
+    # check for us, but has a fixed signature
+    job_id = int(request.match_info['job_id'])
+
+    job_record = await _get_job_record(request.app, batch_id, job_id)
+
+    # effectively the auth check
+    if not job_record:
+        raise web.HTTPNotFound()
```

Comment:
This check is [already covered](https://github.com/hail-is/hail/blob/2863e0eb192c8b845e022f6ed87907eb6f67093a/batch/batch/front_end/front_end.py#L400-L401) in `_get_job_record` (we should add types to the signature), so I think this is dead code.